packages/advanced-logic/test/utils/payment-network/erc20/any-to-erc20-proxy-create-data-generator.ts

```diff
@@ -99,13 +105,19 @@ export const extensionStateCreatedEmpty = {
     events: [
       {
         name: 'create',
-        parameters: {},
+        parameters: {
+          network: 'private',
+          acceptedTokens: [tokenAddress],
+        },
```

Comment:
Previous version + tests made it possible to have no acceptedTokens in these tests, although it is defined as mandatory.